qcfractal/storage_sockets/sqlalchemy_socket.py

```diff
@@ -1413,17 +1413,20 @@ def get_wavefunction_store(
         ----------
         id : List[str], optional
             A list of ids to query
-        include : Dict[str, bool], optional
-            Description
+        include : list/set/tuple of str, default is None
+            The fields to return, default to return all
+        exclude : list/set/tuple of str, default is None
+            The fields to not return, default to return all
+        exclude : Dict
         limit : int, optional
             Maximum number of results to return.
         skip : int, optional
             skip the `skip` results
 
         Returns
         -------
-        TYPE
-            Description
+        Dict with keys: rdata and meta
```

Comment:
Same typing comment from above.

Reply:
Did the same here just like the comment I put above.